Userland/Libraries/LibJS/Runtime/ValueTraits.h

```diff
@@ -23,6 +24,8 @@ struct ValueTraits : public Traits<Value> {
 
         if (value.is_negative_zero())
             value = Value(0);
+        else if (value.is_nan())
```

Comment:
I think a comment here would be helpful for those not familiar with different NaN types

Reply:
Done